mule-extensions-api/src/main/java/org/mule/runtime/extension/internal/loader/validator/ParameterModelValidator.java

```diff
@@ -152,14 +153,22 @@ public void visitObject(ObjectType objectType) {
 
               String fieldName = field.getKey().getName().getLocalPart();
               if (RESERVED_NAMES.contains(fieldName)) {
-                throw new IllegalParameterModelDefinitionException(
-                                                                   format("The field named '%s' [%s] from class [%s] cannot have that name since it is a reserved one",
-                                                                          fieldName, getId(field.getValue()), getId(objectType)));
+                problemsReporter
+                    .addError(new Problem(parameterModel,
+                                          format("The field named '%s' [%s] from class [%s] cannot have that name since it is a reserved one",
+                                                 fieldName, getId(field.getValue()), getId(objectType))));
               }
 
               if (supportsGlobalReferences(field)) {
                 field.getValue().accept(this);
               }
+
+              ExtensionMetadataTypeUtils.getType(field.getValue())
```

Comment:
yeah, this validates that no type used as POJO parameter has the override, but we need the TypeAnnotation for inline groups